cmd/lifecycle-main.go

```diff
@@ -0,0 +1,134 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
+ *
```

Comment:
the file name should be renamed from lifecyle-->ilm to be similar to all other commands.